exchange/c2cx/cli/c2cx.go

```diff
@@ -389,12 +389,18 @@ The public ticker API returns key pricing data for a give currency pair.
 }
 
 func init() {
-	user, err := user.Current()
-	if err != nil {
-		log.Panicf("failed to get the current user. err: %v", err)
+	var config string
+	var err error
+	if os.Getenv("TEST_EXCHANGE_CONFIG") == "" {
```

Comment:
Do not have test specific code here.  Either name the envvar something like `EXCHANGE_API_CONFIG` or add a `-c <config>` CLI flag (https://github.com/skycoin/teller has an example of this)